testUtil/src/main/kotlin/com/commonwealthrobotics/bowlerkernel/testutil/FlowTestUtil.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * This file is part of bowler-kernel.
+ *
+ * bowler-kernel is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * bowler-kernel is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with bowler-kernel.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.commonwealthrobotics.bowlerkernel.testutil
+
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.flow
+
+fun <T> alwaysEmit(vararg t: T) = flow {
+    while (true) {
+        t.forEach { emit(it) }
+        delay(100)
+    }
+}
```

Comment:
What exactly is the purpose of using this rather than `flowOf`? I.e. when using it in the tests, why would we want it to repeat every 100 ms?

Reply:
Sometimes the server receives the clien't message before processing the request I make in the test code, so the test hangs indefinitely. This fixes that by always emitting a response to the request. I can't figure out a better way to do it that involves only emitting the client message once.

Reply:
The proper way to do it (simulating an actual system) would be to make the client wait for the request before sending the response.

Reply:
Yes, but how?